alpine-consul/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM smebberson/alpine-base:1.2.0
 MAINTAINER Scott Mebberson <scott@scottmebberson.com>
 
-ENV CONSUL_VERSION=0.5.2 GOMAXPROCS=2
+ENV CONSUL_VERSION=0.5.2 GOMAXPROCS=2 CONSUL_TEMPLATE_VERSION=0.11.1
+
+# Download and install Consul Template
+ADD https://releases.hashicorp.com/consul-template/${CONSUL_TEMPLATE_VERSION}/consul-template_${CONSUL_TEMPLATE_VERSION}_linux_amd64.zip /tmp/consul-template.zip
```

Comment:
@matthewvalimaki, could you please separate `consul` installation `RUN` from `consul-template` installation? While you're there, could you please remove the `ADD` step for the `consul-template` download and use `curl` instead. This tidy up will more effectively manage the size of the image, and readability/logic of the code.